tests/unit/source_code/test_dependencies.py

```diff
@@ -94,25 +94,6 @@ def test_dependency_resolver_raises_problem_with_unfound_local_notebook_dependen
     ]
 
 
-def test_dependency_resolver_raises_problem_with_non_constant_local_notebook_dependency(mock_path_lookup):
-    notebook_loader = NotebookLoader()
-    notebook_resolver = NotebookResolver(notebook_loader)
-    pip_resolver = PythonLibraryResolver(Whitelist())
-    dependency_resolver = DependencyResolver(pip_resolver, notebook_resolver, [], mock_path_lookup)
-    maybe = dependency_resolver.build_notebook_dependency_graph(Path('root10.py'))
```

Comment:
I don't think this file is used anywhere outside this test? Should it be removed if the test is being removed?

Reply:
Yes